### DISCUSSION TITLE
Create Template in Developer Wiki

### DISCUSSION THREAD

**Discussion by julyarsdadhaniya** [community/community#55895] [New to GitHub]
Hello Github family,

I would like to know by any chance I can create templates in Developer Wiki and used those templates in future and create new pages using standard template.

Appreciate your support.

**Comment by queenofcorgis** [comment]
👋🏼 Hi there - at this time Wikis do not have a template function built in - but I would recommend creating a repo with templates and then copying the raw markdown to create new pages in standardized form.
